.travis.yml

```diff
@@ -21,15 +21,21 @@ install:
     - rm -rf ${CONDA_INSTALL_LOCN}/pkgs && ln -s ${HOME}/cache/pkgs ${CONDA_INSTALL_LOCN}/pkgs
 
     # Now do the things we need to do to install it.
-    - conda create -n test_env --file requirements.txt nose python=${PYTHON} ${EXTRA_DEPS} --yes --quiet
+    - export EXTRA_DEPS='coveralls pip pytest pytest-cov'
+    - conda create -n test_env --file requirements.txt python=${PYTHON} ${EXTRA_DEPS} --yes --quiet
     - source activate test_env
-    - conda list   
-    - python setup.py build_ext install
-    - mkdir not_the_source_root && cd not_the_source_root
```

Comment:
Is it possible to install, rather than have an editable install? I'd like to test that we are including the required files in our distribution, rather than testing we have the necessary files in the source repository.